### ISSUE TITLE
Fixing few issues...

### ISSUE THREAD

**Issue by Lemniscate-SHA-256** [Lemniscate-SHA-256/Neural-.nr#14]
- The Flatten layer output shape includes np.int64() which should be converted to a regular integer.
- The generated TensorFlow code has quotes around the optimizer and loss values that shouldn't be there.
- The training configuration from the input isn't being properly extracted.

**Comment by Lemniscate-SHA-256** [comment]
```
# Remove quotes from optimizer and loss values
        optimizer = str(model_data['optimizer']['value']).strip('"')
        loss = str(model_data['loss']['value']).strip('"')
        code += f"model.compile(loss='{loss}', optimizer='{optimizer}')\n"

        # Extract training configuration
        if 'training' in model_data and model_data['training_config']:
            epochs = model_data['training_config'].get('epochs', 10) # Default to 10 epochs
            batch_size = model_data['training_config'].get('batch_size', 32) # Default to batch size of 32
            code += f"model.fit(data, epochs={epochs}, batch_size={batch_size})\n"
        return code
```
